sklearn/linear_model/tests/test_logistic.py

```diff
@@ -443,11 +444,12 @@ def test_ovr_multinomial_iris():
     n_samples, n_features = train.shape
 
     # Use pre-defined fold as folds generated for different y
-    cv = StratifiedKFold(target, 3)
-    clf = LogisticRegressionCV(cv=cv)
+    cv = StratifiedKFold(3)
+    predefined_folds = list(cv.split(train, target))
```

Comment:
That makes sense, could you please add a brief comment explaining this?
`predefined` is a bit misleading here without good documentation. I'd suggest "precomputed" with a comment that notes that we precompute the split based on the fine-grained iris classes before conflating them.